app/services/socket-io.js

```diff
@@ -26,17 +26,19 @@ export default Service.extend({
   * url meaning multiple requests for the same socket will return the same object.
   */
   socketFor(url, options = {}) {
-    const existingProxy = this.get(`sockets.${normalizeURL(url)}`);
+    const normalizedUrl = normalizeURL(url);
+    const cleanedUrl = normalizedUrl.replace('.', '', 'g');
```

Comment:
Thanks @keithbro for pointing this out. I just updated and will release a new version shortly